tests/foreman/rhci/test_computeresources.py

```diff
@@ -0,0 +1,353 @@
+from robottelo.test import UITestCase
+from robottelo.common import conf
+from robottelo.ui.session import Session
+from robottelo.common.constants import FOREMAN_PROVIDERS
+from robottelo.ui.factory import make_resource
+from ddt import data, ddt
+import time
+
+
+@ddt
+class ComputeResourceTestCase(UITestCase):
+
+    default_org = 'Default Organization'
+    default_loc = 'Default Location'
+
+    vmware_name = conf.properties['vmware.name']
+    vmware_vcenter = conf.properties['vmware.vcenter']
+    vmware_username = conf.properties['vmware.username']
+    vmware_password = conf.properties['vmware.password']
+    vmware_datacenter = conf.properties['vmware.datacenter']
+    vmware_vm_name = conf.properties['vmware.vm_name']
+    vmware_url_name = 'vcenterserver'
+    vmware_dc_locator = 'datacenter_vsphere'
+    vmware_img_name = conf.properties['vmware.img_name']
+    vmware_img_os = conf.properties['vmware.img_os']
+    vmware_img_arch = conf.properties['vmware.img_arch']
+    vmware_img_uname = conf.properties['vmware.img_username']
+    vmware_img_passw = conf.properties['vmware.img_password']
+    vmware_img = conf.properties['vmware.img_image']
+
+    @data(
```

Comment:
This `@data()` decorator has only 1 entry, `subTest` is not needed here too. Better to move the dict with values to some variable at the beginning of the test. Or, if the same dict is passed to all the tests with small changes - consider creating helper function, which will return the dict. Same comment applies to all the data-decorated tests.